### PR TITLE
Adding Features

### PR DESCRIPTION
**Pull request by TayyabKM**
- The project is a single-page application (SPA) built with React
- The data retrieved from the API is stored in the Redux store.
- Every page (the main page and the pages for each item) have a unique route within the SPA.
- The project is deployed and accessible online using Netlify
- Added a link to the online version of app to Readme.md

### CODE REVIEW

File: src/tests/countrySlice.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { formatCountries } from '../redux/country/countrySlice';
```
Comment (BenMukebo) [inline src/tests/countrySlice.test.js:2]: - [ ] Great job so far however according to the project requirements, you could also perform a mocking test data fetching and Redux Store connection. please address this. 👍🏻 

image

**Review by BenMukebo — CHANGES_REQUESTED** [review]
## Hi @TayyabKM, 

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


### To Highlight 🎉

- Git flow is followed. :heavy_check_mark:
- Your app has been well deployed using to Netlify. ✔️ 
- The project contains a `Home` page and a `Detail` page. ✔️ 
- The project is a single page application (SPA) using `react-router-dom` to serve each page without reloading. ✔️
- You have followed  JavaScript best practices. ✔️ 
- Professional README file. :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — CHANGES_REQUESTED** [review]
...

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**